### DISCUSSION TITLE
Multi theme focus

### DISCUSSION THREAD

**Discussion by Vondia** [chakra-ui/panda#1338] [General]
### Subject

Multi- Theme tokens guide

### Description

First of all love the docs super job guys! But in the guide I miss where I have to put the code snippets to make it work. For instance the first config const example. I don't know if i should put this in the panda config or where in my repo. On a lot of places you first show how to make it and than how to use it. Something like this would be an amazing addition to an already great product.

**Comment by anubra266** [comment]
@Vondia I'm not sure I understand your request. Could you shed more light?

**Comment by Vondia** [comment]
> @Vondia I'm not sure I understand your request. Could you shed more light?

I might be in the wrong place it was meant as feedback on the docs just for an improvement. And in alot of code snippets in the docs there is in the top of the snippet also a path of where the code should be written. But that's missing in the guides - Multiple themes part of the docs. 

If this is in the wrong place I will delete this post
